assignment.js

```diff
@@ -66,6 +78,30 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+// declaring variables and test case no. 1
+let numbers = [57, 13, 4, 38]
+let currentLargestNumber = 0
+
+// logic for finding the highest number in give array
+for (i = 0; i < numbers.length; i++) {
```

Comment:
@nhtaylor nice here of using an `if` statement to keep track of the largest number. I used an alternative approach (that violated the requirements) of using `sort()` to solve this. You can check out the approach [here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L86)